app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ContestantsController < ApplicationController
+  def index
+    @contestants = Contestant.all
+  end
+
+  def show
```

Comment:
Just so that you don't end up creating more work for yourself, be careful about what endpoints are necessary and which can be left out. For this IC, you only needed a contestant index page and a project show page. No need to create the others if you don't need em! YAGNI